examples/fib.tau

```diff
@@ -0,0 +1,11 @@
+fib = fn(x) {
+  return if x == 0 {
+    0
+  } else if x == 1 {
+    1
+  } else {
+    fib(x-1) + fib(x-2)
+  }
+}
```

Comment:
Oooh I didn't know you could do this in Tau! Very nice 👌

Reply:
I probably need to write more explicative READMEs 😆.

Reply:
The README is fine, but I _think_ you should highlight this point a bit more, maybe even make it it's own section 😅

Reply:
Thinking about it, with Tau you can also write the function like this, without the explicit return:
```python
fib = fn(x) {
  if x == 0 {
    0
  } else if x == 1 {
    1
  } else {
    fib(x-1) + fib(x-2)
  }
}
```

Reply:
I've got a few other variations I want to try in Tau and port from Monkey Lang just for the fun 😅